pkg/plugin/types/phone.go

```
@@ -0,0 +1,97 @@
+	if err != nil {
+		return "", err
+	}
+
+	if len(decoded) < aes.BlockSize {
+		return "", errors.New("ciphertext too short")
+	}
+
+	iv := decoded[:aes.BlockSize]
+	decrypted := decoded[aes.BlockSize:]
+
+	stream := cipher.NewCFBDecrypter(block, iv)
```

golangci-lint (1.64.8)

SA1019: cipher.NewCFBDecrypter has been deprecated since Go 1.24 and an alternative has been available since Go 1.2: CFB mode is not authenticated, which generally enables active attacks to manipulate and recover the plaintext. It is recommended that applications use [AEAD] modes instead. The standard library implementation of CFB is also unoptimized and not validated as part of the FIPS 140-3 module. If an unauthenticated [Stream] mode is required, use [NewCTR] instead. 

(staticcheck)